commandline/src/main/java/tech/pegasys/eth2signer/commandline/CommandlineParser.java

```diff
@@ -44,9 +49,18 @@ public int parseCommandLine(final String... args) {
     commandLine.setExecutionExceptionHandler(this::handleExecutionException);
     commandLine.setParameterExceptionHandler(this::handleParseException);
 
+    registerConverters(commandLine);
+
     return commandLine.execute(args);
   }
 
+  private void registerConverters(final CommandLine commandLine) {
+    final MetricCategoryConverter metricCategoryConverter = new MetricCategoryConverter();
```

Comment:
nit: not sure how else to do this - but having the converters registered somewhere apart from the Eth2SignerCommand feels odd - i.e. it's the types in Eth2SignerCommand which require converts, so having the logic here (instead of there) - feels messy (happy to be wrong, and not something I'm concerned about, just a smell)

Reply:
It is a bit messy, but the registration needs to happen on the CommandLine. Perhaps passing in convertors to the CommandlineParser would be better? At least then the definitions could stay in the Eth2SignerCommand.

Reply:
Found a better solution using the converter property on the Option annotation in the Eth2SignerCommand